demo/src/app/pages/demo/components/advanced-example-types.component.ts

```diff
@@ -14,77 +15,88 @@ export class AdvancedExamplesTypesComponent {
       name: "Leanne Graham",
       username: "Bret",
       email: "Sincere@april.biz",
-      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu."
+      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu.",
+      custom: '10|20'
```

Comment:
Ok, so you suggest to use an input value that is not an `Object` like `custom: {from: 10, to: 20}` but rather a `string` as all the other cases?